### PR TITLE
remove unused imports

### PR DESCRIPTION
**Pull request by wangborui**
Test Pull Request as a way for code review

### CODE REVIEW

File: src/main/java/contact_management/Contact.java [hunk 1]
```diff
@@ -7,7 +7,6 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
```
Comment (BoTest123) [inline src/main/java/contact_management/Contact.java:9]: Nice imports

File: src/main/java/contact_management/Contact.java [hunk 2]
```diff
@@ -7,7 +7,6 @@
...
-import javax.persistence.FetchType;
```
Comment (BoTest123) [inline src/main/java/contact_management/Contact.java:10]: Good to remove the imports

**Review by BoTest123 — COMMENTED** [review]
Review the comments

**Review by brady-vitrano — APPROVED** [review]
I hate it. But I will allow it

**Pull request merged**